### PR TITLE
Added recipe for librabbitmq

### PR DESCRIPTION
**Pull request by pmlandwehr**
Pinging @ask in case they'd like to be added as a maintainer to the `librabbitmq` builder on [conda-forge](http://conda-forge.github.io), a library of community-built [`conda`](http://conda.pydata.org) packages. The maintenance burden here is light: all testing and releases are built and deployed automatically with CI. Changes to how the package is being built can be controlled by changing the recipe which will be located at https://github.com/conda-forge/librabbitmq-feedstock shortly after this PR is finished and merged. If you aren't interested in helping maintain the build, that's entirely fine too.

### CODE REVIEW

**Comment by pmlandwehr** [comment]
I can probably just kill `import funtests`, but I'd be curious to know why it only fails on `2.7` for windows. Any insights?

**Comment by patricksnape** [comment]
Well it's only running for Python 2.7 right? So it's just failing on Windows. As to why that is - I'm not sure.

**Comment by patricksnape** [comment]
I think [this](https://github.com/celery/librabbitmq/issues/26) is your issue - Windows unsupported?

**Comment by pmlandwehr** [comment]
Ha! Perfect. I'll kill windows for now.

File: recipes/librabbitmq/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+about:
+  home: http://github.com/celery/librabbitmq
+  license: MPL 1.1
+  license_file: LICENSE-MPL-RabbitMQ
+  summary: 'AMQP Client using the rabbitmq-c library.'
+
+extra:
+  recipe-maintainers:
+    - pmlandwehr
```
Comment (jakirkham) [inline recipes/librabbitmq/meta.yaml]: Any interest in helping maintain, @anguslees?

**Comment by jakirkham** [comment]
Thanks @pmlandwehr .

As the CI is really stale, I'm hoping it still passes. 🍀 However, I guess we will find out at the feedstock.

**Pull request merged**